main.py

```diff
@@ -1,7 +1,33 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genre_list = ["Western", "Action", "Dramma"]
```

Comment:
```suggestion
    genres = ["Western", "Action", "Dramma"]
```
_list suffix is redundant